README.md

```diff
@@ -2,36 +2,41 @@
 The Paketo Buildpack for Dynatrace is a Cloud Native Buildpack that contributes the Dynatrace OneAgent and configures it to connect to the service.
 
 ## Behavior
-This buildpack will participate if either of the following conditions is met
+This buildpack will participate if either of the following conditions is met:
 
 * A binding exists with `name` containing `Dynatrace`
 * A binding exists with `type` of `Dynatrace`
 
-**Note**:  While a single binding may match both conditions, you may *not* have multiple bindings that match the conditions above. Multiple Dynatrace service bindings are not supported for a single application.
+**Note**:
+While a single binding may match both conditions, you may *not* have multiple bindings that match the conditions above. Multiple Dynatrace service bindings are not supported for a single application.
 
-**Note**: The binding must include the following required Secret values to successfully contribute Dynatrace
+**Note**:
+The binding must include the following required Secret values to successfully contribute Dynatrace
 
 
-| Key | Value Description
-| -------------------- | -----------
-| `api-url`<br/>  **or** <br/> `environment-id` | The base URL of the Dynatrace API. If not set, `environment-id` must be set. <br/> --- <br/> If `api-url` is not set, a URL is configured in the form: https://<`environment-id`>.live.dynatrace.com/api
-| `api-token` | (Required) The token for communicating with the Dynatrace service.
+| Key                                      | Value Description                                                                                                                                                                                                                    |
+|------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `api-url`, `apiurl`, or `environment-id` | (Required) The base URL of the Dynatrace API. If not set, `environment-id` must be set. <br/> --- <br/> If neither `api-url` or `apiurl` is set, a URL is configured in the form: https://<`environment-id`>.live.dynatrace.com/api. |
```

Comment:
I worry that listing both might create some consternation with users over which one to use (it could just be my brain also). Anyway, while this PR makes it technically accept either one, I think we should continue to promote using `api-url` in this section (i.e. no need to change this table). It's just more direct that way.

I think your `**Note**` at the bottom is sufficient to document that fact that we also permit the keys without dashes. The key is that for *compatibility* we allow it to do both. If you need it to not have the `-` then it'll work, but we recommend using it with the `-`.

Reply:
Just pushed a change - please review and resolve if that works for you.